plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelConfiguratorUI.java

```diff
@@ -230,6 +234,29 @@ private static void setNumberEditStyles(Text text) {
         text.setLayoutData(gdt);
     }
 
+    @Nullable
+    private static DBPAuthInfo promptCredentialsDialog(@NotNull SSHConstants.AuthType type,
+        @NotNull DBWHandlerConfiguration configuration) {
+        final DBPAuthInfo credentialsToConnectTo;
+        try {
+            credentialsToConnectTo = DBWorkbench.getPlatformUI()
+                .promptUserCredentials(SSHUIMessages.model_ssh_dialog_credentials,
```

Comment:
Wrap parameters instead of method call to reduce indentation level

```suggestion
            credentialsToConnectTo = DBWorkbench.getPlatformUI().promptUserCredentials(
                SSHUIMessages.model_ssh_dialog_credentials,
```